Automattic/wp-calypso
# Paid Stats: Adjust payment page styling for WPCOM sites

Related to #81488

## Proposed Changes

* This PR addresses that the current payment page / wizard design might come across as too Jetpack, which might seem a bit inconsistent in a flow for WordPress.com users. It adjusts the colors for WPCOM users as per designs in p1HpG7-orS-p2#comment-65837

* It adjusts the following to match a more WPCOM style:

	* Calypso typography
	* Theme palette used for buttons, links & accents
	* This does not address the "color of the illustration for WordPress.com users" feedback. Which can come in a separate PR

## Testing Instructions



* Load the Calypso live branch
* Go to `stats/purchase/{siteURL}`
* If it is a wpcom simple site, append the flag `flags=stats/paid-wpcom-stats`
* Test on both WPCOM site, and a Jetpack site. Make sure the colors, typography and styling match those in the screenshots below as well as the design iterations at p1HpG7-orS-p2#comment-65837

| step 1.  | personal site | commercial site  |
| ------------- | ------------- | ------------- |
| ![image](https://github.com/Automattic/wp-calypso/assets/30754158/86575a80-0ad7-4fba-aec3-350eeae045bb) | ![image](https://github.com/Automattic/wp-calypso/assets/30754158/8d4322a2-9700-4c89-89c2-0d45badf39ab) | ![image](https://github.com/Automattic/wp-calypso/assets/30754158/97293d66-bffa-4462-88cb-78e6c6515a0b)  |
| ![image](https://github.com/Automattic/wp-calypso/assets/30754158/4de01594-e162-4d8e-8047-a0e9b715fb9c)  | ![image](https://github.com/Automattic/wp-calypso/assets/30754158/1c557844-8228-403d-a2e7-2753bdf8ee88)  | ![image](https://github.com/Automattic/wp-calypso/assets/30754158/e8866166-4504-4f0f-929a-e451c664347f)  |



## Pre-merge Checklist
